helm/chaos-mesh/values.yaml

```diff
@@ -26,6 +26,9 @@ controllerManager:
   nameOverride: ""
   fullnameOverride: ""
 
+  allowedNamespaces: ""
```

Comment:
How about using `controllerManager.env` to manage all environment variables like chaos-dashboard and define a config struct to unmarshal them?

Reply:
Good idea. I'll take a look